pyrsa/model/model.py

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Definition of RSA Model class and subclasses
+@author: jdiedrichsen, heiko
+"""
+
+import numpy as np
+
+
+class Model:
+    """
+    Abstract model class.
+    Defines members that every class needs to have, but does not implement any
+    interesting behavior. Inherit from this class to define specific model
+    types
+    """
+    def __init__(self, name):
+        self.name = name
+        self.n_param = 0
+
+    def predict(self, theta=None):
+        """ Returns the predicted rdm(-vector)
+
+        Args:
+            theta(np.array 1d): the model parameter vector
+
+        Returns:
+            rdm vector
+        """
+        raise NotImplementedError(
+            "Predict function not implemented in used model class!"
+        )
+
+    def fit(self, data):
+        """ fit the model to a RDM object data
+
+        Args:
+            data(RDM object): the RDMs to be fit with the model
+
+        Returns:
+            theta(np.array 1d): parameter vector
+        """
+        return np.array([])
+
+
+class ModelFixed(Model):
+    """
+    Fixed model
+    This is a parameter-free model that simply predicts a fixed RDM
+    It takes a unidimension numpy-vector as an input to define the RDM
+    """
+    # Model Constructor
+    def __init__(self, name, rdm):
```

Comment:
So this would later accept a `pyrsa.Rdms` class instead of the numpy array?

Reply:
I think this is the plan